### PR TITLE
Write new task: Configuring a Pod to Use a Volume for Storage

### PR DESCRIPTION
**Pull request by steveperry-53**
---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1875)

### CODE REVIEW

**Comment by pwittrock** [comment]
I think it is missleading to call an emptydir volume persistent storage.  It disappears when the Pod is deleted.  I think most people would consider Persistent Storage something like a Persistent Disk that has its own lifetime outside of the Pod / Node.

**Comment by steveperry-53** [comment]
@pwittrock I see your point about not calling an emptyDir persistent storage. Some of our existing docs do call this persistent storage, for example http://kubernetes.io/docs/user-guide/production-pods/. But maybe we should come up with another title.
Configuring a Pod to Use a Volume for Storage
Using a Volume to Save State Across Container Restarts
Other ideas?

**Comment by pwittrock** [comment]
Hm, not sure.  @saad-ali?  "Configuring a Pod to Use a Volume for Storage" sounds good

**Comment by saad-ali** [comment]
I agree with @pwittrock. Persistent storage is inaccurate. `Configuring a Pod to Use a Volume for Storage` sounds correct since it does not imply the nature of the volume (which may be ephemeral or persistent).  Existing docs should be corrected as well. @steveperry-53 do you mind making the change or opening an bug to make sure it is fixed.

**Comment by steveperry-53** [comment]
Removed "persistent" from this new topic.
Also removed "persistent" from existing topic in https://github.com/kubernetes/kubernetes.github.io/pull/1931.

**Comment by steveperry-53** [comment]
@pwittrock, OK to merge?

File: docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+---
+---
+
+{% capture overview %}
+
+This page shows how to configure a Pod to use a Volume for storage.
+
+A Container's file system lives only as long as the Container does, so when a
+Container terminates and restarts, changes to the filesystem are lost. For more
+consistent storage that lasts for the life of a Pod, you can use a
```
Comment (pwittrock) [inline docs/tasks/configure-pod-container/configure-volume-storage.md]: nit: 

> For more consistent storage that lasts independent of the container, you can use a...

File: docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+{% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+{% capture steps %}
+
+### Configuring a volume for a Pod
+
+In this exercise, you create a Pod that runs one Container. The Pod has a
+volume that lasts for the life of the Pod, even if the container terminates and
```
Comment (pwittrock) [inline docs/tasks/configure-pod-container/configure-volume-storage.md]: nit: Maybe

> This Pod has a volume that lasts ...

File: docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+restarts. Here is the configuration file for the Pod:
```
Comment (pwittrock) [inline docs/tasks/configure-pod-container/configure-volume-storage.md]: WDYT of:

In this exercise, you create a Pod that runs one Container. The Pod has an` Volume of type EmptyDir`, which lasts for the life of the Pod, even if the container terminates and restarts.  `Some Volumes, such as PersistentVolumeClaims, even continue to exist outside the lifetime of the Pod.`

**Comment by pwittrock** [comment]
Looks good.  Left a couple minor comments.  Merge at your discretion.

File: docs/tasks/configure-pod-container/configure-volume-storage.md (hunk 1 above)
Comment (steveperry-53) [inline docs/tasks/configure-pod-container/configure-volume-storage.md]: Done.

**Pull request merged**